Turkish Translation

kaluju opened the pull request:
reviewing Turkish translation and some minor fixes in de & en

Commit:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: Ic0905f66e03544c549ee6bf0edde3707c528215c

Commit:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: Ic0905f66e03544c549ee6bf0edde3707c528215c

harambasicluka committed:
Translation branch update (#127)

* Implement submission done fragment (#20)

* Added submission done fragment (currently dangling in nav graph)

* fixed leftover javadoc

* Moved QR code scan and registration fragment to submission package (#21)

* Dependency Updates and Gradle Fixes (#34)

* Bump up Gradle to 4.0.0 / 6.1.1

* Bump up NDK

* Bump Up SQLite

* adapt circle config

* Fix #23: Typo in onboarding screen (#31)

Co-authored-by: Jakob Möller <jakob.moeller@sap.com>

* Submission UI Additions (#25)

* Moved camera permission check to dispatcher fragment

* Added illustrations for test result screen

* Updated result fragment to match new design

* Added constraints to status card

* Integrated submission result positive other warning fragment in ui flow

* Added submission done fragment to nav graph

Signed-off-by: Kolya Opahle <k.opahle@sap.com>

* Added ability to dialog helper to create dialog without negative button

* Switched from AlertDialog.Builder to DIalogHelper

* Enable diagnosis key upload to backend (#35)

* Moved camera permission check to dispatcher fragment

* Added illustrations for test result screen

* Updated result fragment to match new design

* Added constraints to status card

* Integrated submission result positive other warning fragment in ui flow

* add TAN fetching to the key submission transaction

* remove unneeded function

* Fix log message for fetching TAN

* Added submission done fragment to nav graph

Signed-off-by: Kolya Opahle <k.opahle@sap.com>

* Added ability to dialog helper to create dialog without negative button

* Switched from AlertDialog.Builder to DIalogHelper

* use actual TAN (authCode) for key submission

set the TAN in the corresponding header field
fixes #8

* request permission to retrieve diagnosis keys

* permission needs to be requested again after dialog has been presented to the users
* trigger transaction to retrieve TAN and upload keys

Co-authored-by: Kolya Opahle <k.opahle@sap.com>

* Removed ExposureNotificationIntentService, the functionality is handled by NotificationHelper (#40)

* Fixing tele tan registration (#45)

* Fixed teleTAN based device registration

Signed-off-by: Kolya Opahle <k.opahle@sap.com>

* allowed excluded tan chars for testing

Signed-off-by: Kolya Opahle <k.opahle@sap.com>

* Revert "allowed excluded tan chars for testing"

This reverts commit 253ede4ea9c8ba2b5e6dd324680eed0f8f7e74c1.

* Release/0.5.5 (#48)

* Release/0.5.5

* Hotfix for resetting Gradle Version (internal pipeline confirmation) (#52)

* Gradle Reset to 0.5.4 (#53)

* Release/0.5.5 (#48)

* Hotfix for resetting Gradle Version (internal pipeline confirmation) (#52)

* Minify and Shrink Disabled (#56)

* Release/0.5.5

* Gradle Reset to 0.5.4 (#53)

* Release/0.5.5 (#48)

* Hotfix for resetting Gradle Version (internal pipeline confirmation) (#52)

* Disable minify / shrink

* Dont obfuscate

* 0.5.6

* Update known issues and fixing typo in readme (#44)

* added strings wip to known issues

* updated readme

Co-authored-by: Muschko <marc.muschko@sap.com>

* Updated strings, introduced more details on app-information screens (#88)

* updated strings for risk card and risk card details; plural strings are todo when finalized

* updated more strings on main / details

* added comments

* added strings for information about, information technical hotline; changed technical hotline layout

* removed hotline test icon

* fixed build issues

* Cleanup pull_request_template.md (#54)

* Update pull_request_template.md

* Removed Link to Contribution Guidelines (added by Github on the right hand side, anyhow)
* Removed Link to issue #41 from title example (this has been linked quite a lot...)

* 0.5.6

* Fix typo

Co-authored-by: Jakob Möller <jakob.moeller@sap.com>
Co-authored-by: harambasicluka <64483219+harambasicluka@users.noreply.github.com>
Co-authored-by: marcmuschko <marc.muschko@sap.com>

* Add Support for lower case characters in the teleTAN screen (#66)

* added support for lower case input in teleTan fragment

Signed-off-by: Kolya Opahle <k.opahle@sap.com>

* Removed notice of case insensitivity from temporary strings.xml

Signed-off-by: Kolya Opahle <k.opahle@sap.com>

Co-authored-by: harambasicluka <64483219+harambasicluka@users.noreply.github.com>

* Correct typos in architecture overview (#67)

Co-authored-by: marcmuschko <marc.muschko@sap.com>

* Bluetooth & connection card (#97)

* introduced settings navigation helper

* added connection and bluetooth stati to main and tracing settings

* moved strings

* removed unnecessary backgroundTint

Co-authored-by: marcmuschko <marc.muschko@sap.com>

* Fix typo in class documentation (#71)

* 0.5.6

* fix typo

Co-authored-by: Jakob Möller <jakob.moeller@sap.com>
Co-authored-by: marcmuschko <marc.muschko@sap.com>

* spelling fixes (#87)

* 0.5.6

* spelling fixes

Co-authored-by: Jakob Möller <jakob.moeller@sap.com>
Co-authored-by: klemens <ka7@github.com>
Co-authored-by: marcmuschko <marc.muschko@sap.com>

* Moves creation of notification channel to app startup (#32)

Co-authored-by: Jakob Möller <jakob.moeller@sap.com>

* Tracing start/stop fix due to missing handling inside tracing (#99)

Co-authored-by: Jakob Möller <jakob.moeller@sap.com>

* Switch the DB Password to use the KeyStore MasterKey (#98)

Signed-off-by: d067928 <jakob.moeller@sap.com>

* Shared prefs ktx shorthand (#38)

* closes #33

* Add Commit Flag to ensure synchronous consistent Updates to Disk

Signed-off-by: d067928 <jakob.moeller@sap.com>

* Add persistance of timestamp and whether user is allowed to submit keys (#74)

* Store timestamp of receiving registration token

* also reset this timestamp when deleting the registration token

* use correct timestamp for display of test time

* replace tracing activation time by test result reception time

* rename TAN request to match context

* set isAllowedToSubmitDiagnosisKeys properly

Co-authored-by: Jakob Möller <jakob.moeller@sap.com>

* set "enabled" for include_settings_switch_row in notifications settings, fixes wrong binding and removed clickable rows (#101)

Co-authored-by: marcmuschko <marc.muschko@sap.com>

* fix #82 (#94)

* fix #82

Co-authored-by: Jakob Möller <jakob.moeller@sap.com>

* Add ShareHelper (#85)

* Add ShareHelper

* Use ShareHelper in MainShareFragment

* Replace the current exposure status with a positive result card when test result is positive (#79)

* Made submission status card style more cross device friendly

Signed-off-by: Kolya Opahle <k.opahle@sap.com>

* Added the positive result submission status card

Signed-off-by: Kolya Opahle <k.opahle@sap.com>

* lint fix

* Updated main fragment documentation

* Added icon for sharing risk status

Signed-off-by: Kolya Opahle <k.opahle@sap.com>

* Updated positive result home screen card text and icons

Signed-off-by: Kolya Opahle <k.opahle@sap.com>

* Added loading indicator to test result fragment (#100)

Co-authored-by: marcmuschko <marc.muschko@sap.com>
Co-authored-by: harambasicluka <64483219+harambasicluka@users.noreply.github.com>

* Adjusted onboarding flow, adjusted contribution file, small fixes in settings (#106)

* changed permission dialog logic for onboarding tracing fragment and onboarding notification fragment

* adjusted contributing file

* added settings notification handling to display value in settings overview

* fixed app crash during bluetooth settings navigation

* corrected wrong value from view model in tracing settings text

* Submission Contact Fragment & StepView Custom View (#102)

* - submission-contact fragment added
- StepEntry custom view added to unify various step entry views
- SimpleStepEntry as a simple implementation of a StepEntry with title and text added

* - apply naming conventions for ids
- use CallHelper to trigger a call
- access views using kotlinx.android.synthetic

Co-authored-by: harambasicluka <64483219+harambasicluka@users.noreply.github.com>

* Notifications when risk value has changed between low and high. (#76)

* Notifications when risk value has changed between low and high.

* Notifications when risk value has changed between low and high.
Fixed wildcard imports.

* Fixed formatting, ktlint issue.

* app foreground/background checking. Notifications are not sent when app is in background

* update comments

* initialize foreground flag as false(valid for background threads before ui starts)

* comments

* included UNDERTERMINED as a low, for initialisation

* fixed icon

* fixed tests

Co-authored-by: Jakob Möller <jakob.moeller@sap.com>

* added onboarding strings (#109)

* disable colored output for ktlint (#105)

Co-authored-by: harambasicluka <64483219+harambasicluka@users.noreply.github.com>
Co-authored-by: Jakob Möller <jakob.moeller@sap.com>

* Limit number of keys to be uploaded to server (#107)

* limit the number of keys to be uploaded to 14

* fix spaces

Co-authored-by: Jakob Möller <jakob.moeller@sap.com>

* Fix white space, informal language and spelling of Docker in Readme (#78)

* 0.5.6

* Fix spelling gonna and docker

* Add missing "to"

Co-authored-by: Jakob Möller <jakob.moeller@sap.com>
Co-authored-by: marcmuschko <marc.muschko@sap.com>

* Update build.gradle (#111)

* Build.Gradle Conflict Resolving (0.8.0) (#112)

* Minify and proguard fix (#113)

Commented out -dontobfuscate

* Circle progress bar (#96)

* circle progress bar impl

* ktlint format

* added new ui

* added circle to contact tracing without binding

* added binding and final icon

* added circle progress to risk card

* added comments

* changed default value for disableText

* adapted views order to designs

* removed testing value

Co-authored-by: Kirill <kirill.sergeev@sap.com>

* TracingStatusHelper unit test (#114)

* TracingStatusHelper unit test

* removed duplicate

* Prevent screenshot of the app for all activities (#108)

* Prevent screenshot of the app for all activities

* fix ktlint

* code smell fix

Co-authored-by: harambasicluka <64483219+harambasicluka@users.noreply.github.com>

* Feature: Bluetooth callback (#115)

* bluetooth callback added

* bound bluetooth callback to the ui

Co-authored-by: Kirill <kirill.sergeev@sap.com>

* Added main overview page to main menu (#120)

* added overview fragment baseline

* main overview draft version completed with build fixes

* checked some final strings again

* Fixed screen orientation to portrait (#121)

* added restriction to portrait and reversed portrait orientation only

* suppress SourceLockedOrientationActivity lint error

* changed local config that a high match can be generated via QR code, integrated it with the UI (#125)

* Feature: Translation preparation (#126)

* renamed old strings and reordered

* app information strings translation tags

* updated translation tags for long texts

* added translation comments to strings

* fixed naming error

* removed empty comment

Co-authored-by: Muschko <marc.muschko@sap.com>

Co-authored-by: Kolya Opahle <k.opahle@sap.com>
Co-authored-by: Jakob Möller <jakob.moeller@sap.com>
Co-authored-by: Michael Keppler <bananeweizen@gmx.de>
Co-authored-by: Thomas Klingbeil <64434904+tklingbeil@users.noreply.github.com>
Co-authored-by: Hee Tatt Ooi <64406309+HeeTattSap@users.noreply.github.com>
Co-authored-by: Muschko <marc.muschko@sap.com>
Co-authored-by: Thomas Kowark <thomas.kowark@sap.com>
Co-authored-by: Robert Scheck <robert-scheck@users.noreply.github.com>
Co-authored-by: Volkmar Vogel <volkmar@vogel.app>
Co-authored-by: ka7 <ka7@la-evento.com>
Co-authored-by: klemens <ka7@github.com>
Co-authored-by: Tim Brüggenthies <tim.brueggenthies@outlook.de>
Co-authored-by: mseele <mseele@gmail.com>
Co-authored-by: Janik Steegmüller <janik.steegmueller@gmail.com>
Co-authored-by: Fabian-K <fabian.kajzar@sap.com>
Co-authored-by: Hee Tatt Ooi <hee.tatt.ooi@sap.com>
Co-authored-by: Matthias Küch <mail@matthias-kuech.de>
Co-authored-by: oemerb <66002424+oemerb@users.noreply.github.com>
Co-authored-by: AlexanderAlferov <64849422+AlexanderAlferov@users.noreply.github.com>
Co-authored-by: Kirill <kirill.sergeev@sap.com>
Co-authored-by: Philipp Woessner <64482866+pwoessner@users.noreply.github.com>

jakobmoellerdev committed:
Remove values-2Q

Signed-off-by: d067928 <jakob.moeller@sap.com>

jakobmoellerdev committed:
Merge remote-tracking branch 'origin/translation' into translation-rebased

# Conflicts:
#	.circleci/config.yml
#	Corona-Warn-App/build.gradle
#	Corona-Warn-App/src/main/java/de/rki/coronawarnapp/service/submission/SubmissionService.kt
#	Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/LocalData.kt
#	Corona-Warn-App/src/main/java/de/rki/coronawarnapp/transaction/SubmitDiagnosisKeysTransaction.kt
#	Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/settings/SettingsTracingFragment.kt
#	Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionDispatcherFragment.kt
#	Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionQRCodeScanFragment.kt
#	Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionResultPositiveOtherWarningFragment.kt
#	Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionTestResultFragment.kt
#	Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/view/CircleProgress.kt
#	Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/DialogHelper.kt
#	Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ProtoFormatConverterExtensions.kt
#	Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/formatter/FormatterRiskHelper.kt
#	Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/security/SecurityHelper.kt
#	Corona-Warn-App/src/main/res/drawable/ic_risk_card_saved_days_14.xml
#	Corona-Warn-App/src/main/res/layout/fragment_main_overview.xml
#	Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml
#	Corona-Warn-App/src/main/res/layout/fragment_submission_done.xml
#	Corona-Warn-App/src/main/res/layout/fragment_submission_test_result.xml
#	Corona-Warn-App/src/main/res/layout/include_risk_card.xml
#	Corona-Warn-App/src/main/res/layout/include_row.xml
#	Corona-Warn-App/src/main/res/layout/include_test_result_step.xml
#	Corona-Warn-App/src/main/res/layout/include_tracing_status_card.xml
#	Corona-Warn-App/src/main/res/values/attrs.xml
#	Corona-Warn-App/src/main/res/values/strings.xml
#	Corona-Warn-App/src/test/java/de/rki/coronawarnapp/util/formatter/TracingStatusHelperTest.kt

jakobmoellerdev committed:
Merge pull request #153 from corona-warn-app/translation-rebased

Translation Rebase

Commit:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: Idf52dece137fdc6e314ed752a8ef6b79b08683b0

harambasicluka committed:
deleted 2Q

harambasicluka committed:
Merge remote-tracking branch 'origin/dev' into translation

harambasicluka committed:
Merge remote-tracking branch 'origin/dev' into translation

Commit:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I63afce1d7307c912b034728d8b78d437974f818d

jakobmoellerdev committed:
Merge branch 'dev' into translation

harambasicluka committed:
excluded translation from lint for dev

jakobmoellerdev committed:
Merge branch 'dev' into translation

harambasicluka committed:
Merge remote-tracking branch 'origin/dev' into translation

harambasicluka committed:
fixed quickBuild errors

harambasicluka committed:
Merge branch 'dev' into translation

harambasicluka committed:
Merge remote-tracking branch 'origin/dev' into translation

harambasicluka committed:
Merge branch 'translation' of https://github.com/corona-warn-app/cwa-app-android into translation

Commit:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I0b2e12f977b0a4bf37716d85e527d8d05168120f

harambasicluka committed:
Merge remote-tracking branch 'origin/dev' into translation

# Conflicts:
#	Corona-Warn-App/src/main/res/values-de/strings.xml
#	Corona-Warn-App/src/main/res/values-en/strings.xml

harambasicluka committed:
fixed local issues

Commit:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: Ied4c086215e1392cb4ac061c9e82760f4757d5b2

harambasicluka committed:
Merge remote-tracking branch 'origin/dev' into translation

# Conflicts:
#	Corona-Warn-App/src/main/res/values-de/strings.xml
#	Corona-Warn-App/src/main/res/values-en/strings.xml

harambasicluka committed:
fixes

harambasicluka committed:
Merge remote-tracking branch 'origin/dev' into translation

# Conflicts:
#	Corona-Warn-App/src/main/assets/terms_en.html
#	Corona-Warn-App/src/main/res/values-de/strings.xml
#	Corona-Warn-App/src/main/res/values-en/strings.xml

harambasicluka committed:
resolved merge conflicts

harambasicluka committed:
Merge branch 'dev' into translation

jakobmoellerdev committed:
Merge branch 'dev' into translation

harambasicluka committed:
Merge remote-tracking branch 'origin/dev' into translation

# Conflicts:
#	Corona-Warn-App/src/main/res/values-en/strings.xml

Commit:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I0dbb47c30c1f19b7533e953e349a3145694db366

harambasicluka committed:
Merge remote-tracking branch 'origin/dev' into translation

harambasicluka committed:
Merge remote-tracking branch 'origin/dev' into translation

harambasicluka committed:
Merge remote-tracking branch 'origin/dev' into translation

Commit:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: If73a3358a2939902b8df4e491fb7b3beb626f8fc

harambasicluka committed:
Merge branch 'dev' into translation

Commit:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I101e87820c63ecb7c40b044ac0bc2e05331b352b

harambasicluka committed:
Merge remote-tracking branch 'origin/dev' into translation

# Conflicts:
#	Corona-Warn-App/src/main/res/values-de/strings.xml
#	Corona-Warn-App/src/main/res/values-en/strings.xml

harambasicluka committed:
changed default value for strings

harambasicluka committed:
added https to english faq links

harambasicluka committed:
fixed error during merge

marcmuschko committed:
Merge branch 'dev' into translation

jakobmoellerdev committed:
Merge branch 'dev' into translation

Commit:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I8b27c238c3387741363b0dc353711994217801b6

Commit:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I9bc5d20688a11bfd5fd4a481b3c36143d4caba64

DerVogel2020 commented on Corona-Warn-App/src/main/res/values-en/strings.xml:
```diff
@@ -535,7 +535,7 @@
     <string name="information_terms_title">"Terms of Use"</string>
     <!-- XHED: Page headline for terms of use information page -->
-    <string name="information_terms_headline">"Content"</string>
+    <string name="information_terms_headline">"Inhalt"</string>
```
This file is /values-en/strings.xml -> the german "Inhalt" is wrong

harambasicluka replied:
fixed

kaluju committed:
Merge branch 'dev' into translation

service-tip-git committed:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: Ib3f20898335ded0101af92c452c766c329946611

service-tip-git committed:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I9b5bbd9b5b5d44029f694b7afca8f7251b48e86a

kaluju reviewed (commented):
pull request is ready for review

harambasicluka commented on Corona-Warn-App/src/main/res/values-en/strings.xml:
```diff
@@ -185,7 +185,7 @@
     <string name="main_about_body">"Here you can find answers to frequently asked questions about the Corona-Warn-App. You will be forwarded to an external website."</string>
     <!-- NOTR -->
-    <string name="main_about_link">"https://www.bundesregierung.de/corona-warn-app-faq-englisch"</string>
+    <string name="main_about_link">"https://www.bundesregierung.de/corona-warn-app-faq"</string>
```
@kaluju that's wrong for my understanding, my should use the english FAQ link or have I missed something?

harambasicluka commented on Corona-Warn-App/src/main/res/values-en/strings.xml:
```diff
@@ -135,7 +135,7 @@
     </plurals>
     <!-- XTXT: risk card - tracing active for x out of 14 days -->
-    <string name="risk_card_body_saved_days">"%1$s of 14 days active"</string>
+    <string name="risk_card_body_saved_days">"%1$s of  days active"</string>
```
Thank's for your feedback, both issues you found where fixed :)

harambasicluka committed:
added privacy, technical and terms html file with english content

harambasicluka commented on Corona-Warn-App/src/main/res/values-tr/strings.xml:
```diff
@@ -0,0 +1,1007 @@
+    <!-- XACT: describes illustration -->
+    <string name="information_privacy_illustration_description">"Bir kadın cihazında Corona-Warn-App\'i kullanıyor. Kılıftaki kilit, şifrelenmiş verilerin sembolüdür."</string>
+    <!-- XTXT: Path to the full blown privacy html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_privacy_html_path">"privacy_en.html"</string>
```
should be "translated" to `privacy_tr.html` to be consistent with the other languages, for now I added the english versions to this files.

kaluju replied:
implemented

harambasicluka commented on Corona-Warn-App/src/main/res/values-tr/strings.xml:
```diff
@@ -0,0 +1,1007 @@
+    <!-- XACT: describes illustration -->
+    <string name="information_terms_illustration_description">"Bir kişi ekranında uzun bir metin olan bir cihaz tutuyor. Metnin yanında kullanım koşullarını kabul ettiğini belirten bir sembol olarak tik işareti bulunuyor."</string>
+    <!-- XTXT: Path to the full blown terms html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_terms_html_path">"terms_en.html"</string>
```
should be "translated" to `terms_tr.html` to be consistent with the other languages, for now I added the english versions to this files.

kaluju replied:
implemented

harambasicluka commented on Corona-Warn-App/src/main/res/values-tr/strings.xml:
```diff
@@ -0,0 +1,1007 @@
+    <!-- XACT: describes illustration -->
+    <string name="information_technical_illustration_description">"Bir kişi ekranında uzun bir metin olan bir cihaz tutuyor. Metnin yanında yasal bildirimlerin sembolü olarak bir terazi bulunuyor"</string>
+    <!-- XTXT: Path to the full blown legal html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_technical_html_path">"technical_en.html"</string>
```
should be "translated" to `technical_tr.html` to be consistent with the other languages, for now I added the english versions to this files.

kaluju replied:
implemented

harambasicluka commented on Corona-Warn-App/src/main/res/values-tr/strings.xml:
```diff
@@ -0,0 +1,1007 @@
+    <!-- XTXT: main, explains faq on card -->
+    <string name="main_about_body">"Bu bölümde, Corona-Warn-App hakkında sık sorulan soruların yanıtlarını bulabilirsiniz. Harici bir web sitesine yönlendirileceksiniz."</string>
+    <!-- NOTR -->
+    <string name="main_about_link">"https://www.bundesregierung.de/corona-warn-app-faq"</string>
```
A turkish translation isn't available yet?

harambasicluka committed:
Merge remote-tracking branch 'origin/translation' into translation

schlotter commented on Corona-Warn-App/src/main/res/values-de/strings.xml:
```diff
@@ -934,17 +935,15 @@
     <string name="errors_not_enough_device_storage">"Sie haben nicht genug Speicherplatz."</string>
     <!-- XTXT: error dialog - detailed text if there is error with Google API  -->
-    <string name="errors_communication_with_api">"Fehler bei Kommunikation mit Google API"</string>
+    <string name="errors_communication_with_api">"Fehler bei der Kommunikation mit dem Google API"</string>
```
API is of course an English acronym, so no rule for the declension of it exists in German, but I think it is more common to decline API as a feminine noun. Thus, the sentence should be:
_Fehler bei der Kommunikation mit **der** Google API_

sapkra replied:
You're right, but I think it would be best to use a word that is not technical, that all people can understand. e.g. "Schnittstelle" instead of "API".

Additionally I was wondering why the German translation is changed in a PR that should only add the Turkish translation. But that is of course another topic.

harambasicluka replied:
@sapkra our translation tool only updates the translation branch and sometimes some minor changes are then also introduced in a PR like this.

kaluju replied:
@schlotter @sapkra Thank you for your feedback. I changed the German to "Schnittstelle".

duchampdev commented:
Please consider #792 before merging this PR.

duchampdev committed:
include Turkish translation (#792)

I added Turkish to resConfigs for properly including the new translation into the build.

harambasicluka committed:
Merge branch 'dev' into translation

harambasicluka commented:
#792 is merged. Thank you for your contribution @duchampdev

harambasicluka committed:
replaced NOTR tag with XTXT to make "main_about_link" translatable

harambasicluka committed:
Merge branch 'translation' of https://github.com/corona-warn-app/cwa-app-android into translation

service-tip-git committed:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I01f4b04ec61a334e0b25815b03e2953e5031e0d9

harambasicluka committed:
Merge remote-tracking branch 'origin/translation' into translation

service-tip-git committed:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I71fcec61630c5d99e009eeb1341fd37bdd0c55e9

harambasicluka committed:
Merge branch 'dev' into translation

Fabian-K committed:
string placeholders required for background priority added (#797)

Fabian-K committed:
additional strings for explanatory card added (#798)

harambasicluka committed:
add turkish privacy & terms

service-tip-git committed:
[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I3708d2af77b48642eee7a6562cffd935327ed875

harambasicluka committed:
added correct faq links and lorem ipsum text

harambasicluka committed:
updated default strings with values-en/strings

harambasicluka committed:
fixed issues with new translation process
